src/scope.ml

```diff
@@ -130,6 +197,35 @@ let scope : mode -> sig_state -> env -> p_term -> tbox = fun md ss env t ->
     let c = Pervasives.ref (-1) in
     fun () -> Pervasives.incr c; Printf.sprintf "#%i" Pervasives.(!c)
   in
+  let add_implicits_outermost : p_term -> p_term = fun t ->
+    match t.elt with
+      | (P_Appl(_, _) as e)
+      | (P_Iden(_, _) as e)   ->
+        (* split the function symbol and its given arguments *)
+        let (f, args) = split_fun_args (none e) in
+        (* get the formal parameters of f *)
+        let params = get_params_implicitness_of_id ss env f in
+        (* adds the implicit arguments *)
+        let fullArgs = add_implicit_args params args in
+          add_args_pterm f fullArgs
+      | x                   -> none x
+  in
+  (* let add_implicits : p_term -> p_term = fun t ->
```

Comment:
What is this? Please, remove.

Reply:
It was a temporary version of the code (the PR was updated with my commits to my repo).